modules/ROOT/pages/mule-deployment-model.adoc

```diff
@@ -0,0 +1,26 @@
+= Mule App Deployment Model
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, esb, deploy, architecture
+
+A Mule Project for deployment is a *Mule Application*. A Mule runtime instance supports multiple applications running independently within the same Mule container. This allows additional advantages such as:
+
+* Running multiple applications side-by-side
+* Clear boundaries for operations on what a Mule application is
+* Mule can monitor your applications and will reload configuration changes automatically
+* Applications can depend on different library versions, even if they would conflict before
+* Multiple versions of an application can exist side by side
```

Comment:
side by side should have hyphens between the words.

Reply:
Yes. They can run at the same time. Updated.